From a19066254cabaebcee6bb0ee14389f4222a31963 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Fri, 20 Feb 2015 12:23:36 -0600
Subject: [PATCH 014/278] arch/arm64: Correct GIC Physical Address in Axxia XLF

Simulation had the wrong physical address for the GIC. This patch
changes it to the correct address, which matches the chip.

Signed-off-by: John Jacques <john.jacques@intel.com>
---
 arch/arm64/boot/dts/intel/axc67xx.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/intel/axc67xx.dtsi b/arch/arm64/boot/dts/intel/axc67xx.dtsi
index 32358a2..03fcd17 100644
--- a/arch/arm64/boot/dts/intel/axc67xx.dtsi
+++ b/arch/arm64/boot/dts/intel/axc67xx.dtsi
@@ -67,8 +67,8 @@
 		#interrupt-cells = <3>;
 		#address-cells = <0>;
 		interrupt-controller;
-		reg = <0x80 0x10010000 0x10000>,
-		      <0x80 0x10100000 0x200000>;
+		reg = <0x80 0x10000000 0x10000>,
+		      <0x80 0x10200000 0x200000>;
 		interrupts = <GIC_PPI 9 IRQ_TYPE_LEVEL_HIGH>;
 	};
 
-- 
1.8.5.2.233.g932f7e4

